site/src/components/SplashPage.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import splashPageStyles from '@site/src/components/css/SplashPage.module.css';
+// import splashPageStyles from '@site/src/css/pages/SplashPage.module.scss';
```

Comment:
Can we remove this if its just commented out? 
```suggestion
```

Reply:
yes, it's a remnant from when I was removing those component specific css files